src/card.css

```diff
@@ -0,0 +1,123 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body,
+p {
+  margin: 0;
+
+  --border-color: #f3f3f3;
+  --text-color: #060b35;
+  --sub-text-color: #616071;
+  --button-color: #00acdc;
+  --white-color: #fff;
+  --border-radius: 5px;
+}
+
+.card {
+  max-width: 200px ;
+  max-height: 408px;
+  border: 1px solid var(--border-color);
+  border-radius: var(--border-radius);
+  padding: 32px 16px 16px;
+
+  display: flex;
+  box-sizing: border-box;
```

Comment:
move `box-sizing` to `*` selector to avoid duplicates

Reply:
check all such cases

Reply:
* not fixed

try to fix all of the previous comments before making re-request.

If you have any questions - feel free to ask them in fe_chat